modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/item/test/FileEntryContentDashboardItemTest.java

```diff
@@ -0,0 +1,702 @@
+
+		Assert.assertEquals(
+			"example.jpg",
+			versionableContentDashboardItem.getTitle(LocaleUtil.getDefault()));
+	}
+
+	@Test
+	public void testGetTypeLabel() throws Exception {
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				_getVersionableContentDashboardItem(
+					1,
+					ServiceContextTestUtil.getServiceContext(
+						_group.getGroupId()));
+
+		Assert.assertEquals(
+			"Document",
+			versionableContentDashboardItem.getTypeLabel(
+				LocaleUtil.getDefault()));
+	}
+
+	@Test
+	public void testGetUserId() throws Exception {
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				_getVersionableContentDashboardItem(
+					1,
+					ServiceContextTestUtil.getServiceContext(
+						_group.getGroupId()));
+
+		Assert.assertEquals(
+			TestPropsValues.getUserId(),
+			versionableContentDashboardItem.getUserId());
+	}
+
+	@Test
+	public void testGetUserName() throws Exception {
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				_getVersionableContentDashboardItem(
+					1,
+					ServiceContextTestUtil.getServiceContext(
+						_group.getGroupId()));
+
+		Assert.assertEquals(
+			"Test Test", versionableContentDashboardItem.getUserName());
+	}
+
+	@Test
+	public void testIsShowContentDashboardItemVersions() throws Exception {
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				_getVersionableContentDashboardItem(
+					1,
+					ServiceContextTestUtil.getServiceContext(
+						_group.getGroupId()));
+
+		Assert.assertTrue(
+			versionableContentDashboardItem.isShowContentDashboardItemVersions(
+				_getMockHttpServletRequest()));
+	}
+
+	@Test
+	public void testIsViewableWithLayoutPageTemplateEntry() throws Exception {
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				_getVersionableContentDashboardItem(
+					1,
+					ServiceContextTestUtil.getServiceContext(
+						_group.getGroupId()));
+
+		Assert.assertTrue(
+			versionableContentDashboardItem.isViewable(
+				_getMockHttpServletRequest()));
+	}
+
+	@Test
+	public void testIsViewableWithoutLayoutPageTemplateEntry()
+		throws Exception {
+
+		FileEntry fileEntry = _dlAppLocalService.addFileEntry(
+			RandomTestUtil.randomString(), TestPropsValues.getUserId(),
+			_group.getGroupId(), DLFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			"example.jpg",
+			MimeTypesUtil.getExtensionContentType(ContentTypes.IMAGE_JPEG),
+			new byte[0], null, null,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+
+		VersionableContentDashboardItem<FileEntry>
+			versionableContentDashboardItem =
+				(VersionableContentDashboardItem<FileEntry>)
+					_contentDashboardItemFactory.create(
+						fileEntry.getFileEntryId());
+
+		Assert.assertFalse(
+			versionableContentDashboardItem.isViewable(
+				_getMockHttpServletRequest()));
+	}
+
+	@Test
+	public void testViewVersionURL() throws Exception {
```

Comment:
Following Liferay naming convention, the name of the test should be getViewVersionsURL.

Reply:
done, thanks for your feedback @cgoncas :)